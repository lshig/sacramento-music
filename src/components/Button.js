import React from 'react';

export default function Button() {
  return (
    <div className="code-info">
      <a
        className="button__large liz-logo"
        href="https://lshig.github.io"
        rel="noopener noreferrer"
        target="_blank"
      >
        <span>Liz Shigetoshi</span>
      </a>
      <a
        className="button__large"
        href="https://github.com/lshig/sacramento-music"
        rel="noopener noreferrer"
        target="_blank"
      >
        <i className="fa fa-github" />
        <span>View on GitHub</span>
      </a>
    </div>
  );
}
