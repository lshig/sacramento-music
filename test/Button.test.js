import { shallow } from 'enzyme';
import React from 'react';
import toJson from 'enzyme-to-json';
import Button from '../src/components/Button';

describe('Button', () => {
  describe('basic', () => {
    const wrapper = shallow(<Button />);

    test('renders', () => {
      expect(toJson(wrapper)).toMatchSnapshot();
    });
  });
});
