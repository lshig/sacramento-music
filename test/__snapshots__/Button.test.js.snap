// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button basic renders 1`] = `
<div
  className="code-info"
>
  <a
    className="button__large liz-logo"
    href="https://lshig.github.io"
    rel="noopener noreferrer"
    target="_blank"
  >
    <span>
      Liz Shigetoshi
    </span>
  </a>
  <a
    className="button__large"
    href="https://github.com/lshig/sacramento-music"
    rel="noopener noreferrer"
    target="_blank"
  >
    <i
      className="fa fa-github"
    />
    <span>
      View on GitHub
    </span>
  </a>
</div>
`;
